app/validators/schema_validator.py

```diff
@@ -41,6 +42,7 @@ def validate(self):
             self.schema_validator.validate(self.schema_element)
             return {}
         except ValidationError as e:
+            print(e)
```

Comment:
Needs to be removed.

Reply:
removing print statement left in error. 👍